packages/components/src/Combobox/components/ComboboxContent/ComboboxContent.tsx

```diff
@@ -19,17 +20,33 @@ interface ComboboxContentProps {
    * actions
    */
   readonly children: ReactElement | ReactElement[];
+
+  /**
+   * pre selected option
+   * @default ""
+   * @optional
+   * @type string
+   * */
+  readonly preSelectedOption?: string;
 }
 
 export function ComboboxContent(props: ComboboxContentProps): JSX.Element {
+  const { preSelectedOption } = props;
```

Comment:
very much a style/preference thing, but to me the value of doing `props: ComboboxContentProps` is that when you use it via `props.preSelectedOption` it becomes very evident that it came from a prop rather than a hook or a local variable!

Reply:
ah yes! good point we should stay consistent here 👍🏻